Makefile

```
@@ -2,7 +2,7 @@ BINARY_NAME=terramaid
 VERSION=v1
 GO=go
 
-default: build
+default: help
```

checkmake (0.3.2)

[warning] Target "default" should be declared PHONY.

(phonydeclared)